src/org/scada_lts/dao/event/EventDAO.java

```diff
@@ -0,0 +1,928 @@
+				+"e."+COLUMN_NAME_ID+"=?";
+	
+	private static final String EVENT_ACT ="" +
+			"update "
+				+"events set "
+				+ COLUMN_NAME_ACT_TS+"=?, "
+				+ COLUMN_NAME_ACT_USER_ID+"=?, "
+				+ COLUMN_NAME_ALTERNATE_ACK_SOURCE+"=? "
+		  + "where "
+				+ COLUMN_NAME_ID+"=? and "
+				+ "("+COLUMN_NAME_ACT_TS+" is null or "+COLUMN_NAME_ACT_TS+" = 0) ";
+	
+	public static final String EVENT_FILTER_ACTIVE=" "
+			+"e."+ COLUMN_NAME_RTN_APPLICABLE+"=? and e."+ COLUMN_NAME_RTN_TS+" is null";
+	
+	private static final String EVENT_SELECT_WITH_USER_DATA=""
+			+"select "
+				+ "e."+COLUMN_NAME_ID+", "
+				+ "e."+COLUMN_NAME_TYPE_ID+", "
+				+ "e."+COLUMN_NAME_TYPE_REF_1+", "
+				+ "e."+COLUMN_NAME_TYPE_REF_2+","
+				+ "e."+COLUMN_NAME_ACTIVE_TS+","
+				+ "e."+COLUMN_NAME_RTN_APPLICABLE+", "
+				+ "e."+COLUMN_NAME_RTN_TS+","
+				+ "e."+COLUMN_NAME_RTN_CAUSE+", "
+				+ "e."+COLUMN_NAME_ALARM_LEVEL+", "
+				+ "e."+COLUMN_NAME_MESSAGE+", "
+				+ "e."+COLUMN_NAME_ACT_TS+", "
+				+ "e."+COLUMN_NAME_ACT_USER_ID+", "
+				+ "u."+COLUMN_NAME_USER_NAME+","
+				+ "e."+COLUMN_NAME_ALTERNATE_ACK_SOURCE+", "
+				+ "ue."+COLUMN_NAME_SILENCED+" "
+			+ "from "
+				+ "events e " 
+				+ "left join users u on e."+COLUMN_NAME_ACT_USER_ID+"=u.id "
+				+ "left join userEvents ue on e."+COLUMN_NAME_ID+"=ue."+COLUMN_NAME_EVENT_ID;
+				
+	private static final String EVENT_FILTER_FOR_DATA_POINT=""+
+			"e."+COLUMN_NAME_TYPE_ID+"=" + EventType.EventSources.DATA_POINT+" and "
+		  + "e."+COLUMN_NAME_TYPE_REF_1+"=? and "
+		  + "ue."+COLUMN_NAME_USER_ID+"=? "
+		  + "order by e."+COLUMN_NAME_ACTIVE_TS+" desc";
+	
+	private static final String EVENT_FILTER_TYPE_REF_USER = ""
+			+"e."+COLUMN_NAME_TYPE_ID+"=? and "
+			+"e."+COLUMN_NAME_TYPE_REF_1+"=? and "
+			+"ue."+COLUMN_NAME_USER_ID+"=? and "
+			+"((e."+COLUMN_NAME_ACT_TS+" is null or e."+COLUMN_NAME_ACT_TS+"=0) or (e."+COLUMN_NAME_RTN_APPLICABLE+"=? and e."+COLUMN_NAME_RTN_TS+" is null and e."+COLUMN_NAME_ALARM_LEVEL+" > 0))"
+			+"order by e."+COLUMN_NAME_ACT_TS+ " desc";
+	
+	private static final String EVENT_FILTER_TYPE_USER = ""
+			+"e."+COLUMN_NAME_TYPE_ID+"=? and "
+			+"ue."+COLUMN_NAME_USER_ID+"=? and "
+			+"((e."+COLUMN_NAME_ACT_TS+" is null or e."+COLUMN_NAME_ACT_TS+"=0) or (e."+COLUMN_NAME_RTN_APPLICABLE+"=? and e."+COLUMN_NAME_RTN_TS+" is null and e."+COLUMN_NAME_ALARM_LEVEL+" > 0))"
+			+"order by e."+COLUMN_NAME_ACT_TS+ " desc";
+	
+	private static final String EVENT_FILTER_USER = ""
+			+"ue."+COLUMN_NAME_USER_ID+"=? and "
+			+"((e."+COLUMN_NAME_ACT_TS+" is null or e."+COLUMN_NAME_ACT_TS+"=0) "
+			+"order by e."+COLUMN_NAME_ACT_TS+ " desc";
+	
+	private static final String EVENT_COMMENT_SELECT = ""
+			+"select "
+				+ "uc."+COLUMN_NAME_USER_ID+", "
+				+ "u."+COLUMN_NAME_USER_NAME+", "
+				+ "uc."+COLUMN_NAME_TIME_STAMP+","
+				+ "uc."+COLUMN_NAME_COMMENT_TEXT+" "
+		    + "from "
+		    	+ "userComments uc left join users u on uc."+COLUMN_NAME_USER_ID+" = u."+COLUMN_NAME_ID
```

Comment:
This statement creates "u.idwhere" instead of "u.id where". You should add a space between COLUMN_NAME_ID and "where".